src/apis/setActiveResult.js

```diff
@@ -1,6 +1,17 @@
-import actions from 'actions';
+import core from 'core';
 
-export default store => (result, index) => {
-  store.dispatch(actions.setActiveResult(result));
-  store.dispatch(actions.setActiveResultIndex(index));
-};
\ No newline at end of file
+export default (result, index) => {
+  if (!result && index >= 0) {
+    // core doesn't have concept of active result index. In previous implementation
```

Comment:
I improved this text a bit, but I disagree that these comments should not be in code. I've ran in to code in WV where we have 5 arguments, but current implementation only uses one and it's object rather than number. There is no explanation anywhere of the functionality and took me ages and asking from people to figure out, that we only have that for backward compatibility. I these kinda cases having few line explanation would have saved me a lot of time

Reply:
I agree with you. I spent a bunch of time on the same issue. And unfortunately, JS doesn't have method overload.

I will check with Matt with we can roll this out in the next major version.